device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/twin/DirectMethodPayload.java

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.azure.sdk.iot.device.twin;
+
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonElement;
+import lombok.AllArgsConstructor;
+
+@AllArgsConstructor
```

Comment:
Does this include both public and private fields?

Reply:
> 

Yes, it generates an all-args constructor which requires one argument for every field in the class.